com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -578,6 +597,11 @@ internal void Clear(CameraRenderType cameraType)
             for (int currIndex = blockRanges[blockIndex]; currIndex < endIndex; ++currIndex)
             {
                 var renderPass = m_ActiveRenderPassQueue[currIndex];
+                renderPass.debugMaterialIndex = debugMaterialIndex;
```

Comment:
It would be nice if a struct could be set instead or have function for renderPass.

Reply:
I have an incoming change that moves all of these debug-variables into a separate class.

Reply:
Yes - absolutely, I've got an upcoming update to the PR that does just that.